pkg/bootkube/create.go

```diff
@@ -48,7 +48,7 @@ func CreateAssets(manifestDir string, timeout time.Duration) error {
 		return fmt.Errorf("Failed to create assets: %v", err)
 	}
 
-	UserOutput("Spawning self-hosted control plane...\n")
+	UserOutput("Spawning self-hosted control plane and checkpointing components...\n")
```

Comment:
Yeah, I definitely get that, and I appreciate it. My only real concern is that it's very possible we have bootkube install other services as well that are outside of the control plane, so once that starts happening we either continue to not distinguish, or we come up with a more generic way of labeling and displaying these components.